src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs

```diff
@@ -475,7 +473,7 @@ ResourceManifest BuildManifest()
 
             manifest
                 .DefineScript("monaco")
-                .SetAttribute("data-tenant-prefix", _pathBase)
+                .SetAttribute("data-tenant-prefix", _pathBase.Value)
```

Comment:
In other context e.g. to generate an href the browser will be clever enough to do the right url escaping, but here it is a data-attribute that will be used by a script, so it is important that we pass a well escaped pathBase.

This by using `_pathBase.ToString()` or as before just `_pathBase` because `ToString()` is then implicitly called because the method parameter is a string. `_pathBase.Value` is the original string which is nor url escaped.

Reply:
@Skrypt you requested that that I add `.Value` here due to inconcestancy in other places. Are you good with reverting this change per @jtkech request and explanation?

Reply:
He's right for that one.

Reply:
Updated